data/human/free worlds intro.txt

```diff
@@ -66,91 +75,85 @@ mission "Pact Recon 0"
 		dialog phrase "generic waypoint on visit"
 	on complete
 		payment 25000
-		dialog
-			`A different female militia officer approaches you when you land on <planet>, and gives you <payment> in exchange for your scanner logs of the southern systems. "My name is Freya Winters," she says. "I'll be in touch when we've got a new assignment for you."`
+		dialog `When you land on <planet>, you're approached by Freya Winters. She gives you <payment> in exchange for your scanner logs of the Southern systems. "Thank you, Captain. I pray the pirates didn't give you too much trouble," she says. "I'll be in touch when we've got a new assignment for you, assuming you're still interested."`
```

Comment:
Why has she functionally raced ahead of you to be here?

In fact this would be a great place to introduce a different officer, who takes the place of the Officer Winters reference later on. Someone who remains localised to their base of operations as Freya's offsider.

After all, Freya is a busy and important person; it would make sense she'd have a direct aid who can physically insert themselves into conversations where information flow from base to Freya or vice versa would happen.

Reply:
The player is tasked with flying around the south before returning to Glaze. If the player needs to go from point A to points B, C, D, and E before returning to Glaze, then it isn't exactly "racing ahead" for Freya to go from point A to Glaze.

I'm not looking to introduce new FW characters to the bunch, as then I need to continually worry about them throughout the rest of the campaign. And if the character is unnamed, then we're just back to square one, as having an unnamed officer is what it was before. The purpose of bringing Freya into this mission instead of having this be some unnamed officer is to give the player more interaction with Freya during the intro, as right now this is the only place the player can interact with her up until the player actually joins the Free Worlds.

Reply:
I would argue that having a known entity who stays at base for someone else's benefit (not yours) is pretty safe re worrying about them throughout the story, and it's hard to do things like create side-plots and other things if there's nothing to build off of in terms of minor characters etc.

I hear you though, it's a fair point if you want to say that that's out of scope for this given the range of things you're addressing.

Reply:
I will say that I did think about how Freya and Tomek are now no longer always interacted with on Glaze and Longjump respectively, which is what I think MZ was going for when he had them introduced further into their mission chains instead of at the start, but I'd prefer the appearance of the Council, or at least certain members of it, as being leaders of the people, interacting with them on various planets within Pact/FW territory, as opposed to always being in ivory towers.

Reply:
To be fair, that is more or less how the Republic campaign has been planned out; so that's a valid contrast if that's what you want to aim for.